lib/DevTools.js

```diff
@@ -1,6 +1,12 @@
-import {connectViaExtension} from 'remotedev';
 import _ from 'underscore';
 
+const {connectViaExtension} = process.env.NODE_ENV === 'production' ? require('remotedev') : {
+    connectViaExtension: () => ({
+        init: () => {},
+        send: () => {},
+    }),
+};
+
 class DevTools {
```

Comment:
nab, Where did this file come from and what does it do? 😂 Is there documentation for how to use it available?

Reply:
I think the branch was just not up to date with the latest `main`. Outdated now